browse-handler/java/org/vufind/solr/handler/BrowseRequestHandler.java

```diff
@@ -123,7 +123,7 @@ public HeadingsDB(String path)
     {
         try {
             this.path = path;
-            normalizer = NormalizerFactory.getNormalizer ();
+            normalizer = NormalizerFactory.getNormalizer();
```

Comment:
I see that you've made a variety of minor style fixes here and there. Are you using a particular tool to do this, or just eyeballing as you go? It might be beneficial to automate style fixing if appropriate tooling exists. (That's obviously a project for another day, but just something this made me think of).

Reply:
I used `astyle` with the options or One True Brace Style, as we settled on earlier. Those are now documented in the README.md. Not certain how that space got in there or survived the previous style fixing.

Reply:
Thanks! If you run `astyle --dry-run` does it return a different value depending on whether or not changes are needed? If so, we could probably code an astyle run into Travis so the build will fail if the style is out of whack....

Reply:
Alas, no. We could parse the output. Here's a sample:

```
usagi:vufind-browse-handler tod$ astyle --mode=java --style=1tbs --dry-run -U -H -I -R browse-handler/java/org/vufind/solr/handler/client/solrj/\*
------------------------------------------------------------
Directory  browse-handler/java/org/vufind/solr/handler/client/solrj/*
------------------------------------------------------------
Unchanged  BrowseRequest.java
Formatted  BrowseResponse.java
```
Could perhaps grep for `^Formatted`, but that seems a touch fragile. If we're really motivated, we could contact the maintainer for possible approaches.

Reply:
Grepping for `^Formatted` is probably not a bad place to start, and we could worry about bothering the maintainer if it eventually proved to be fragile. At a glance, I think it looks like opportunities for false positives are fairly limited, so it's really just a question of whether the output format gets changed in a future release.